datatableview/views/base.py

```diff
@@ -76,7 +76,6 @@ def get_ajax(self, request, *args, **kwargs):
         """ Called in place of normal ``get()`` when accessed via AJAX. """
 
         datatable = self.get_datatable()
-        datatable.configure()
```

Comment:
Earlier, configured datatable in ajax but not in first get. That kept 'ordering' from going into the <th>. So I moved this line to below, so datatable configure either way.